examples/storage-disk/src/main.rs

```diff
@@ -45,58 +47,93 @@ impl User {
             ctx,
         }
     }
+
+    fn print_disk_info(&self) {
+        let disk_info = self.disk.borrow().info();
+        log_info!(
+            self.ctx,
+            "Disk info: capacity = {}, used space = {}, free space = {}",
+            disk_info.capacity,
+            disk_info.used_space,
+            disk_info.free_space
+        )
+    }
 }
 
 impl EventHandler for User {
     fn on(&mut self, event: Event) {
         cast!(match event.data {
             Start {} => {
-                log_debug!(self.ctx, "Test #0: Reading 3 bytes... should be OK");
-                self.requests.insert(self.disk.borrow_mut().read(3, self.ctx.id()), 0);
-
-                log_debug!(self.ctx, "Test #1: Then trying to read 6 bytes... should fail");
-                self.requests.insert(self.disk.borrow_mut().read(6, self.ctx.id()), 1);
-
-                log_debug!(self.ctx, "Used space: {}", self.disk.borrow().used_space());
-
-                log_debug!(self.ctx, "Test #2: Writing 4 bytes... should be OK");
-                self.requests.insert(self.disk.borrow_mut().write(4, self.ctx.id()), 2);
-
-                log_debug!(self.ctx, "Used space: {}", self.disk.borrow().used_space());
-
-                log_debug!(self.ctx, "Test #3: Writing 2 more bytes... should fail");
-                self.requests.insert(self.disk.borrow_mut().write(2, self.ctx.id()), 3);
-
-                log_debug!(self.ctx, "Used space: {}", self.disk.borrow().used_space());
+                self.ctx.emit_self(Ticker {}, 0.);
+            }
+            Ticker {} => {
+                let time = self.ctx.time();
+                if time.eq(&0.) {
```

Comment:
Почему не просто так?

```suggestion
                if time == 0. {
```